### DISCUSSION TITLE
How to shift focus to the right (when I am on a stacked window)

### DISCUSSION THREAD

**Discussion by louwers** [pop-os/shell#1548] [Q&A] [answered]
```
[ tab 1 | tab 2] [ other window ]
```

tab 1 is active

How can I go directly to other window with the keyboard?

Super + right selects tab 2 first.

**Comment by jacobgkau** [comment, marked as answer]
This is not possible in the current implementation of Pop!_Shell. See https://github.com/pop-os/shell/issues/1531 for some related discussion.

This may be possible in the upcoming COSMIC Epoch reimplementation of Pop!_Shell (with tiling/stacking handled by the Rust-based COSMIC compositor rather than as a GNOME Shell extension.) See https://github.com/pop-os/cosmic-epoch/issues/39 for a description of the currently proposed behavior from a member of the UX team (subject to change).

In short, the currently proposed new behavior would look the same when switching focus, but would keep the last-active tab of the stack active when executing the focus change (so in your example, you would still see Tab 2 while you navigate to Other Window, but once you release the keys/press Enter and Other Window is focused, Tab 1 would come back to the top.)

**Reply by louwers** [reply]
Thank you for your detailed reply!
